functions/snippets/src/main/java/HelloBackground.java

```diff
@@ -14,32 +14,23 @@
  * limitations under the License.
  */
 
-// [START functions_helloworld_background]
-import com.google.gson.Gson;
-import com.google.gson.JsonObject;
-import java.io.IOException;
-import java.io.PrintWriter;
-import java.util.logging.Logger;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
+// [START functions_background_helloworld]
 
-public class HelloBackground {
+import com.google.cloud.functions.BackgroundFunction;
+import com.google.cloud.functions.Context;
+import com.google.cloud.functions.HttpRequest;
+import java.util.logging.Logger;
 
-  // Use GSON (https://github.com/google/gson) to parse JSON content.
-  private Gson gsonParser = new Gson();
+public class HelloBackground implements BackgroundFunction<HttpRequest> {
+  private static final Logger LOGGER = Logger.getLogger(HelloBackground.class.getName());
```

Comment:
static finals loggers should refer to the class itself: 
```suggestion
  private static final Logger LOGGER = Logger.getLogger(HelloBackground.class);
```

Reply:
(Same in other samples)

Reply:
I don't think there is such a method as `Logger.getLogger(Class)`, is there?

Reply:
Nope - I get a syntax error.  😦

Reply:
My mistake - I was confusing java.util.logging with `[slf4j](http://www.slf4j.org/apidocs/org/slf4j/LoggerFactory.html#getLogger(java.lang.Class))`. The sample guide does say we should be using java.util.logging, so disregard this.